docs/data/material/discover-more/related-projects/related-projects.md

```diff
@@ -7,63 +7,74 @@ Developers from the community have built some excellent supplemental tools for w
+  - Templates, authentication, database integration, subscription payments, and more.
 
+- [ScaffoldHub](https://v2.scaffoldhub.io/scaffolds/react-material-ui?partner=1):
   - Tool for building web applications.
   - Choose your framework and library (React with Material UI).
   - Choose your database (SQL, MongoDB or Firestore).
   - Model your database and application with the intuitive GUI.
   - Generate your application, including a complete scaffolded backend.
   - Preview your application online and download the generated code.
 
-- [Divjoy](https://divjoy.com?via=material-ui):
-
-  - Create a Material UI app in minutes.
-  - Templates, authentication, database integration, subscription payments, and more.
-
-## Free tools
-
-### Admin frameworks
-
-- [React-admin](https://github.com/marmelab/react-admin): A front-end framework for building data-driven applications running in the browser on top of REST/GraphQL APIs.
-- [Refine](https://github.com/refinedev/refine): An open source, headless React meta-framework for the rapid development of web applications.
-
-### Theming tools
+## Theming tools
 
 - [Material palette generator](https://m2.material.io/inline-tools/color/): The official Material Design 2 palette generator tool.
 
-### Supplementary components
+## Complementary components
 
-#### Layout
+### Layout
 
 - [MUI Treasury Layout](https://mui-treasury.com/?path=/docs/layout-v6-introduction--docs): Components to handle the overall layout of a page. Check out examples such as [a legacy.reactjs.org clone](https://mui-treasury.com/?path=/story/layout-v6-app-react-legacy--react-legacy).
 
-#### Image
+### Image
 
 - [mui-image](https://github.com/benmneb/mui-image): The only Material UI image component to satisfy the Material Design 2 guidelines for loading images.
 
-#### Chips input
+### Chips input
 
 - [mui-chips-input](https://github.com/viclafouch/mui-chips-input): A chips input designed for use with Material UI.
 
-#### Phone number input
+### Phone number input
 
 - [mui-tel-input](https://github.com/viclafouch/mui-tel-input): A phone number input designed for use with Material UI, built with [libphonenumber-js](https://www.npmjs.com/package/libphonenumber-js).
 
-#### One-time password input
+### One-time password input
 
 - [mui-otp-input](https://github.com/viclafouch/mui-otp-input): A one-time password input designed for use with Material UI.
 
-#### File input
+### File input
 
 - [mui-file-input](https://github.com/viclafouch/mui-file-input): A file input designed for use with Material UI.
 
-#### Color input
+### Color input
 
 - [mui-color-input](https://github.com/viclafouch/mui-color-input): A color input designed for use with Material UI, built with [TinyColor](https://tinycolor.vercel.app/).
 
-#### Rich text editor
+### Rich text editor
 
 - [mui-tiptap](https://github.com/sjdemartini/mui-tiptap): A customizable Material UI-styled WYSIWYG rich text editor, built with [Tiptap](https://tiptap.dev/).
+
+### Form
```

Comment:
I guess we might want to duplicate the content?

Reply:
GitHub has a smart way to handle this: https://github.com/github/docs/blob/c20e5a5cb73d4100b635810c927f0c71a7ef30d3/content/desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github-in-github-desktop.md?plain=1#L18. Love it